testsuite/cluster-test/src/instance.rs

```diff
@@ -167,6 +174,22 @@ impl Instance {
         &self.peer_name
     }
 
+    pub fn validator_index(&self) -> String {
+        let re = VAL_REGEX.as_ref().unwrap();
+        if let Some(cap) = re.captures(&self.peer_name) {
+            if let Some(cap) = cap.get(1) {
+                return cap.as_str().to_string();
+            }
+        }
+        let re = FULLNODE_REGEX.as_ref().unwrap();
+        if let Some(cap) = re.captures(&self.peer_name) {
+            if let Some(cap) = cap.get(1) {
+                return cap.as_str().to_string();
+            }
+        }
+        panic!("Failed to find peer_id for : {}", self.peer_name)
```

Comment:
This message is very confusing
Mb "Failed to parse peer name {} into validator_index"?